webportal/src/app/create_item/components/create_data/basic_information.jsx

```diff
@@ -0,0 +1,256 @@
+      (isEmpty(itemDescription.useInTheCode.value)
+        ? ''
+        : `### Use in the code\n\n${itemDescription.useInTheCode.value}\n\n`) +
+      (isEmpty(itemDescription.relatedProject.value)
+        ? ''
+        : `## Related project\n\n${itemDescription.relatedProject.value}\n\n`)
+    );
+  }
+
+  function AdvancedDescriptionTextField({ field, rows }) {
+    return (
+      <TextField
+        label={field.label}
+        multiline={true}
+        rows={rows}
+        value={field.value}
+        onChange={(event, newValue) => {
+          field.value = newValue;
+        }}
+        placeholder={field.placeholder}
+        styles={textStyles}
+      />
+    );
+  }
+
+  AdvancedDescriptionTextField.defaultProps = {
+    rows: 2,
+  };
+
+  AdvancedDescriptionTextField.propTypes = {
+    field: PropTypes.object,
+    rows: PropTypes.number,
+  };
+
+  return (
+    <BasicInformationArea>
+      <Stack {...columnProps}>
+        <TextField
+          label='Title'
+          required
+          value={state.itemObject.name}
+          onChange={(event, newValue) => {
+            state.itemObject.name = newValue;
+            setState({ itemObject: state.itemObject });
+            forceUpdate();
+          }}
+          errorMessage={
+            errorMessage && isEmpty(state.itemObject.name)
+              ? 'Please enter name here.'
+              : undefined
+          }
+          styles={textStyles}
+        />
+        <TextField
+          label='Data url'
+          value={state.itemObject.dataUrl}
+          onChange={(event, newValue) => {
+            state.itemObject.dataUrl = newValue;
+            setState({ itemObject: state.itemObject });
+            forceUpdate();
+          }}
+          placeholder='Please add the data url here'
+          styles={textStyles}
+        />
+        <TextField
+          label='Short summary'
+          description='(No more than 100 characters)'
+          required
+          value={state.itemObject.summary}
+          onChange={(event, newValue) => {
+            state.itemObject.summary = newValue;
+            setState({ itemObject: state.itemObject });
+            forceUpdate();
+          }}
+          errorMessage={
+            errorMessage && isEmpty(state.itemObject.summary)
+              ? 'Please enter summary here.'
+              : undefined
+          }
+          styles={textStyles}
+        />
+        <TextField
+          label='Description'
+          required
+          multiline={true}
+          rows={2}
+          value={itemDescription.description}
+          onChange={(event, newValue) => {
+            itemDescription.description = newValue;
+            forceUpdate();
+          }}
+          errorMessage={
+            errorMessage && isEmpty(itemDescription.description)
+              ? 'Please enter description here.'
+              : undefined
+          }
+          placeholder={`Please add the description of ${
```

Comment:
remove, only add placeholder if it is really needed

Reply:
Done.